src/types/mod.rs

```diff
@@ -248,6 +250,8 @@ pub(crate) mod float;
 mod frame;
 pub(crate) mod frozenset;
 mod function;
+#[cfg(Py_3_9)]
```

Comment:
I am planning on keeping the Python version, `pyo3-ffi` has an empty module but for this case it is more tricky as there are tests, the imports would need to be conditional, etc.